src/view/components/TabManager/index.tsx

```diff
@@ -0,0 +1,136 @@
+import * as React from 'react';
```

Comment:
is it confusing that TabManager is in components when it isn't really a component? maybe it would be suited more in another directory like lib or something. And maybe if it was renamed to something like `TabManagerGenerator` so it is clear that it is a function that generates a component? 
ie. if i hadn't talked to you about this, I would've thought that you would try to extend a tab manager component

Reply:
also, are components in the view folder just UI components? I thought this was the case but maybe I've been misunderstanding this whole time haha

Reply:
Yeah that's true, I'll look into changing it. I originally intended on extending which is why I named it this way

Reply:
Yeah they should just be UI components